tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -5376,7 +5371,7 @@ def test_create_event_source_validation(
         snapshot.match("error", response)
 
     @markers.aws.validated
-    @pytest.mark.skipif(is_v2_esm, reason="ESM v2 validation for Kafka poller only works with ext")
+    @pytest.mark.skip(reason="ESM v2 validation for Kafka poller only works with ext")
```

Comment:
nit: uhh this test probably shouldn't live here lol

Reply:
yep, summarized in follow-up tasks as discussed